include/EqualNeighborHeuristic.h

```diff
@@ -17,6 +17,7 @@ class EqualNeighborHeuristic : public MeshAmalgamation {
   bool belongToCluster(libMesh::Elem *elem,
                        libMesh::Elem *neighbor_elem) override;
   void setTolerance(double tol) { _tol = tol; }
+  virtual double calculateMetrics(libMesh::Elem *elem) override;
```

Comment:
This is a minor thing, but I think there are some small benefits to these appearing in the same order in each derived header, and possibly before the methods that are unique to this class?